doc/install.md

```diff
@@ -1,39 +1,75 @@
+The easiest way to install Livepeer is by downloading the pre-built `livepeer` and `livepeer_cli` executables.
 
-![image](https://user-images.githubusercontent.com/257909/58923612-3709a800-86f5-11e9-838b-6202f296bce8.png)
+### Downloading / installing the software
+
+1. Go to [Livepeer's release page on Github](https://github.com/livepeer/go-livepeer/releases).
+2. Under "Assets", download the `.tar.gz` files for your operating system - `darwin` for MacOS, `linux` for Linux.
+2. Untar them with `tar -xzf livepeer-linux-amd64.tar.gz` or `tar -xzf livepeer-darwin-amd64.tar.gz`
+
+### Running the software
+
+You can then run the `livepeer` binary with the following simple command.
+```
+./livepeer-linux-amd64/livepeer -broadcaster
+```
+This creates an `RTMP` endpoint on `127.0.0.1:1935` and an `HLS` media server on `127.0.0.1:8935`.
+
+### Basic test of the installation
+
+You can serve `RTMP` content into the endpoint using the following command:
+```
+ffmpeg -re -f lavfi -i testsrc=size=360x640:rate=30,format=yuv420p -f lavfi -i sine -threads 1 -c:v libx264 -b:v 4000000 -preset ultrafast -x264-params keyint=30 -strict -2 -c:a aac -f flv rtmp://127.0.0.1:1935/test_signal
+```
+You can query the `HLS` content coming from the media server using:
+```
+curl http://127.0.0.1:8935/stream/test_signal.m3u8
+```
+If you receive the following message, your basic `livepeer` node is running:
+```
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-STREAM-INF:PROGRAM-ID=0,BANDWIDTH=4000000,RESOLUTION=360x640
+test_signal/source.m3u8
+```
+
+For regular builds published by Livepeer's automated build process, see below.
 
-### Option 2: Build from source
-You can also build the executables from scratch.
+## Option 2: Build from source
 
-&ensp; 1\. If you have never set up your Go programming environment, do so according to Go's [Getting Started Guide](https://golang.org/doc/install).
+You can also build your own executables from source code.
 
-&ensp; 2\. You can fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal.
+### Pre-requisites and Setup
 
-&ensp; 3\. Make sure you have the necessary libraries installed:
+&ensp; 1\. Install Go, using Go's [Getting Started Guide](https://golang.org/doc/install).
 
-* Linux: `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev`
+&ensp; 2\. Make sure you have the necessary libraries installed:
+
+ * Linux: `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git`
 
  * OSX: `brew update && brew install pkg-config autoconf gnutls`
 
-&ensp; 4\. You need to install `ffmpeg` as a dependency.  Run `./install_ffmpeg.sh`.  This will install the dependencies in `~/compiled`.  You need to have `pkg-config` installed.
+&ensp; 3\. Fetch the code running `go get github.com/livepeer/go-livepeer/cmd/livepeer` in terminal. This will download the software to `/src/github.com/livepeer/go-livepeer` in your `go` install folder.
```

Comment:
Is it still the case that it'll  download to `/src/...` with go 1.13+ (the current required version) ? It's been a while since I've tried a fresh install.

Reply:
Yes, `go get` is now considered legacy and still has the same behaviour of putting new packages in `GOPATH/src/<import-path>` 

I believe we _could_  update this to `git clone` given we support go modules ?

Reply:
I personally use the `git clone` approach, but not sure what is easier for users who might not be familiar with git / golang. Either one is okay with me.

Reply:
I have no objections to anything proposed.

Reply:
I also use the `git clone` approach so let's switch to that !

Reply:
OK, feel free to change it to what you want.

Reply:
I'll do it in a separate PR since it's a forked branch.

Reply:
I think this is resolved here: https://github.com/livepeer/go-livepeer/commit/303ef94b3a16e9aaafe76490e11c5e53535c6871